Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "LiveServer"
 uuid = "16fef848-5104-11e9-1b77-fb7a48bbb589"
 authors = ["Jonas Asprion <jonas.asprion@gmx.ch", "Thibaut Lienart <tlienart@me.com>"]
-version = "0.9.2"
+version = "0.10.0"
```

Comment:
I think this could be 0.9, there is nothing breaking here, right?

Reply:
well now `0.9.2` is HTTP 0.8, 0.9 but this one is just HTTP 1; also we bumped up the minimum req on Julia 1.6, that's why I thought it'd make sense to make it a 0.10

Reply:
Okay, I think it only matters if you plan on doing more patch releases supporting HTTP 0.8 and 0.9. Perhaps make this 1.0 instead then? Then you have more flexibility for future changes like this since you have three numbers to play around with.

Reply:
I think 1.0.0 once we're happy this is solid would make sense yes :+1: